.github/workflows/assets.yml

```diff
@@ -1,26 +1,36 @@
-name: Compile assets
+name: Compile dist
 
 on:
-  push:
-    paths:
-      - 'www/assets/**'
-  pull_request:
+  pull_request_review:
+    types:
+      - submitted
     paths:
       - 'www/assets/**'
 
 jobs:
-  assets:
+  build:
+    if: github.event.review.state == 'approved'
     runs-on: ubuntu-16.04
```

Comment:
Does the workflow use the old Ubuntu version on purpose?

Reply:
No,, I think that's just a copy-and-paste artifact of me looking at some guides.

Reply:
`ubuntu-latest` is recommended.